scripts/exposure-scanning/scanExposures.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-console */
+/* eslint-disable import/extensions */
+import parseArgs from './args/index.js';
+import { fetchPr, fetchIssue } from './fetch/index.js';
+import { scanPr, scanIssue } from './scan/index.js';
+import { patchPr, patchIssue } from './patch/index.js';
+
+const scanExposures = async () => {
+  const { repo, flag, id, regexString } = parseArgs(process.argv);
+
+  const regex = new RegExp(regexString, 'gi');
+
+  if (flag === '-pr') {
+    const reqBody = {
+      owner: 'bbc',
+      repo,
+      prId: id,
+    };
+
+    const pr = await fetchPr(reqBody);
+
+    const scannedPr = await scanPr(pr, regex);
+
+    await patchPr(reqBody, scannedPr);
```

Comment:
Will this call the github api to patch, even when there is nothing to redact? We could hit rate limiting issues if so. It would be good to only call the api when necessary if this is the case.

Reply:
Really good point. I can definitely make this change 👍